release/gcb/helm_charts.sh

```diff
@@ -29,6 +29,10 @@ cp -R "./istio-${CB_VERSION}/install" "$WORK_DIR/istio/install"
 
 pushd "$WORK_DIR"
     git clone -b "${CB_BRANCH}" https://github.com/istio/cni.git
+    if [ "${CB_PIPELINE_TYPE}" = "monthly" ]; then
+      sed -i "s|hub: gcr.io/istio-release|hub: ${CB_DOCKER_HUB}|g" cni/deployments/kubernetes/install/helm/istio-cni/values.yaml
+      sed -i "s|tag: master-latest-daily|tag: ${CB_VERSION}|g" cni/deployments/kubernetes/install/helm/istio-cni/values.yaml
```

Comment:
is this always master-latest-daily? I would image at some point we have this be `release-1.2-latest-daily` or something

Reply:
It is always master-latest-daily. 
https://github.com/istio/cni/blob/release-1.2/deployments/kubernetes/install/helm/istio-cni/values.yaml